Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTest_Customer.m

```diff
@@ -150,25 +150,59 @@ - (void)testCustomerLogin
 
 - (void)testCustomerLogout
 {
-	[OHHTTPStubs stubUsingResponseWithKey:@"testCustomerLogin" useMocks:[self shouldUseMocks]];
+	[OHHTTPStubs stubUsingResponseWithKey:@"testCustomerLogout" useMocks:[self shouldUseMocks]];
 	
 	XCTestExpectation *expectation = [self expectationWithDescription:NSStringFromSelector(_cmd)];
-	[self.client loginCustomerWithCredentials:[self credentialsForLogin] callback:^(BUYCustomer *customer, NSString *token, NSError *error) {
+	[self.client logoutCustomerID:self.customer.identifier.stringValue callback:^(BUYStatus status, NSError * _Nullable error) {
 		
-		[OHHTTPStubs stubUsingResponseWithKey:@"testCustomerLogout" useMocks:[self shouldUseMocks]];
+		XCTAssertNil(error);
+		XCTAssertEqual(status, 204);
 		
-		[self.client logoutCustomerID:customer.identifier.stringValue callback:^(BUYStatus status, NSError * _Nullable error) {
-			
-			XCTAssertNil(error);
-			XCTAssertEqual(status, 204);
-			
-			[expectation fulfill];
-		}];
+		[expectation fulfill];
 	}];
 	
 	[self waitForExpectationsWithTimeout:10 handler:nil];
 }
 
+#pragma mark - Orders -
+
+- (void)testCustomerOrders
+{
+	[OHHTTPStubs stubUsingResponseWithKey:@"testCustomerGetOrders" useMocks:[self shouldUseMocks]];
+	
+	XCTestExpectation *expectation = [self expectationWithDescription:NSStringFromSelector(_cmd)];
+	[self.client getOrdersForCustomerWithID:self.customer.identifier.stringValue callback:^(NSArray<BUYOrder *> * _Nullable orders, NSError * _Nullable error) {
+		
+		XCTAssertNotNil(orders);
+		XCTAssertNil(error);
+		
+		XCTAssertTrue(orders.count > 0);
+		XCTAssertTrue([orders.firstObject isKindOfClass:[BUYOrder class]]);
+		
+		[expectation fulfill];
+	}];
+	
+	[self waitForExpectationsWithTimeout:10.0 handler:nil];
+}
+
+- (void)testCustomerOrderWithID
+{
+	[OHHTTPStubs stubUsingResponseWithKey:@"testCustomerGetOrder" useMocks:[self shouldUseMocks]];
+	
+	XCTestExpectation *expectation = [self expectationWithDescription:NSStringFromSelector(_cmd)];
+	[self.client getOrderWithID:self.customerOrderIDs.firstObject customerID:self.customer.identifier.stringValue callback:^(BUYOrder * _Nullable order, NSError * _Nullable error) {
+		
+		XCTAssertNotNil(order);
```

Comment:
It's line 195 that is superfluous, since the class of `nil` is not right, and it should be very clear from the test failure what is wrong.